test/index.html

```diff
@@ -13,21 +13,6 @@
 		/* global WCT */
 		'use strict';
 		WCT.loadSuites([
```

Comment:
There's a bunch of tests missing from here that weren't running before.  In this PR, that's `mixin/simple-entity-mixin-test.html`, `AlignmentsCollectionEntity.html`, and `FilesHomeEntity.html`.